components/schemes/Typography/examples/MDCTypographySchemeFontListExampleViewController.m

```diff
@@ -0,0 +1,198 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCTypographySchemeFontListExampleViewController.h"
+
+#import "MaterialColorScheme.h"
+#import "MaterialTypographyScheme.h"
+
+@interface HairlineSeparatorView : UITableViewCell
+
+@end
```

Comment:
I think it might be easier if you customize the existing UITableView cell dividers — did you try this and run into a roadblock that led to this solution?

Reply:
Spoke offline. This is being added as a section divider.